src/equations/shallow_water_two_layer_2d.jl

```diff
@@ -635,25 +443,17 @@ end
 
 
 """
-    flux_es_fjordholm_etal(u_ll, u_rr, orientation_or_normal_direction,
-                           equations::ShallowWaterTwoLayerEquations1D)
+    flux_es_ersing_etal(u_ll, u_rr, orientation_or_normal_direction,
+                           equations::ShallowWaterTwoLayerEquations2D)
 Entropy stable surface flux for the two-layer shallow water equations. Uses the entropy conservative 
-flux_fjordholm_etal and adds a Lax-Friedrichs type dissipation dependent on the jump of entropy
-variables. 
-
-Further details are available in the paper:
-- Ulrik Skre Fjordholm (2012)
-Energy conservative and stable schemes for the two-layer shallow water equations.
-[DOI: 10.1142/9789814417099_0039](https://doi.org/10.1142/9789814417099_0039)
-It should be noted that the equations are ordered differently and the
-designation of the upper and lower layer has been changed which leads to a slightly different
-formulation.
+[`flux_wintermeyer_etal`](@ref) and adds a Lax-Friedrichs type dissipation dependent on the jump of 
+entropy variables. 
 """
-@inline function flux_es_fjordholm_etal(u_ll, u_rr,
+@inline function flux_es_ersing_etal(u_ll, u_rr,
                                         orientation_or_normal_direction, 
                                         equations::ShallowWaterTwoLayerEquations2D)   
```

Comment:
```suggestion
@inline function flux_es_ersing_etal(u_ll, u_rr,
                                     orientation_or_normal_direction, 
                                     equations::ShallowWaterTwoLayerEquations2D)   
```

Reply:
Done in 7d75332d1eb1396dc1d0070ec2f53c8d2501853c